### DISCUSSION TITLE
Resizing images

### DISCUSSION THREAD

**Discussion by BahneGork** [oleeskild/obsidian-digital-garden#338] [Q&A]
As far as i can tell from trial and error, using the default resize argument in an image link wont work. Using it on embedded images in a table breaks the image link and only shows "!200". 

embedding images in a table at normal size results in the tables columns not showing, only the first column with images.

**Comment by starryxoxo** [comment]
Hello, it's been two years, I'll just reply my workaround just in case anyone may stumble across the same issue.

You can workaround this issue by using the other syntax for images:
![alt|size](file.jpg)

**Reply by JonoGraf** [reply]
Not sure why but can't see the image for the alternate syntax

I've tried using html syntax and still no dice
(I'm using ITS theme and complete noob to this!)
